test/unit/org/apache/cassandra/utils/streamhist/StreamingTombstoneHistogramBuilderTest.java

```diff
@@ -124,10 +125,8 @@ public void testNumericTypes() throws Exception
         builder.update(2);
         builder.update(2, Integer.MAX_VALUE); // To check that value overflow is handled correctly
         TombstoneHistogram hist = builder.build();
-        Map<Integer, Integer> asMap = asMap(hist);
-
-        assertEquals(1, asMap.size());
-        assertEquals(Integer.MAX_VALUE, asMap.get(2).intValue());
+        Map<Long, Integer> asMap = asMap(hist);
+        assertEquals(Integer.MAX_VALUE, asMap.get(2l).intValue());
```

Comment:
Nit: `2L` instead of `2l`, some typographies make it difficult to distinguish between `l` and `1`, and this makes some style checks complain.

Reply:
Ah you see, on mine it reads perfectly well but yes, `L` should be better in other typographies. Will do